dags/openshift_nightlies/manifest.yaml

```diff
@@ -1,12 +1,20 @@
-versionAliases:
+versions:
   - version: 4.7
     alias: previous
+    releaseStream: 4.7.0.0-nightly
+    baremetalReleaseStream: latest-4.7
   - version: 4.8
     alias: stable
+    releaseStream: 4.8.0-0.nightly
+    baremetalReleaseStream: latest-4.8
   - version: 4.9
     alias: next
+    releaseStream: 4.9.0-0.nightly
+    baremetalReleaseStream: latest-4.9
   - version: "4.10"
     alias: future
+    releaseStream: 4.10.0-0.nightly
+    baremetalReleaseStream: latest-4.10
 
 dagConfig:
   schedules:
```

Comment:
i thought the plan was to move schedules under each variant so that different variants can have different schedules.

Reply:
this is the schedules we want to use:
    sdn-control-plane: "0 12 * * 1,3,5"
    sdn-data-plane: "0 12 * * 1"
    sdn-medium: "0 12 * * 1"
    sdn-large: "0 12 15 * *"
    ovn-control-plane: "0 12 * * 1,3,5"
    ovn-data-plane: "0 12 * * 1"
    ovn-medium: "0 12 * * 1"
    ovn-large: "0 12 15 * *"

Reply:
yeah the code in `manifest.py` supports adding it at the variant level, I just haven't added them yet.

Reply:
added